oxide-auth/src/primitives/issuer.rs

```diff
@@ -287,9 +287,13 @@ impl<G: TagGrant> Issuer for TokenMap<G> {
         self.set_duration(&mut grant);
         let until = grant.until;
 
-        let next_usage = self.usage.wrapping_add(1);
+        let next_usage = self.usage.wrapping_add(2);
+
         let new_access = self.generator.tag(self.usage, &grant)?;
-        let new_key: Arc<str> = Arc::from(new_access.clone());
+        let new_refresh = self.generator.tag(self.usage.wrapping_add(1), &grant)?;
```

Comment:
From a semantic standpoint this is fine. From a security standpoint it could be cleaner to more clearly show that you're using two different `usage` tags and that the `next_usage` will also differ. I was thinking either a comment or assigning the two cases to variables first.

Reply:
@HeroicKatora Made a change now, assigned tags to a variable between token generation. Let me know if this is what you had in mind.